cmd/gonic/gonic.go

```diff
@@ -36,6 +36,7 @@ func main() {
 	confCachePath := set.String("cache-path", "", "path to cache")
 	confDBPath := set.String("db-path", "gonic.db", "path to database (optional)")
 	confScanInterval := set.Int("scan-interval", 0, "interval (in minutes) to automatically scan music (optional)")
+	confScanWatcher := set.Bool("scan-watcher", false, "whether to watch file system for new music and rescan (optional)")
```

Comment:
how about scan-watcher-enabled for consistency?

Reply:
Done.